app/views/question_flow/applicant.html.slim

```diff
@@ -26,14 +26,10 @@
                                           legend: { text: t(".over_60.legend") },
                                           hint: { text: t(".over_60.hint") }
 
-    = form.govuk_radio_buttons_fieldset(:partner,
-            legend: { text: t(".partner.legend") },
-            hint: -> { content_for(:partner_hint) })
-      = form.govuk_radio_button :partner,
-              true,
-              label: { text: t("generic.yes_choice") },
-              link_errors: true
-      = form.govuk_radio_button :partner, false, label: { text: t("generic.no_choice") }
+    = form.govuk_collection_radio_buttons(:partner,
```

Comment:
question - so are we using the `govuk_collection_radio_buttons` class/element, rather than the partial added to `govuk_radio_buttons_fieldset`, because these options are just a generic "Yes/No" (and we have nice helper methods)?? 

I'm guessing that if the radio button options are used more than once we'd want to stick them here: **app/helpers/forms_helper.rb** and use `govuk_collection_radio_buttons`. Otherwise, we'd use `govuk_radio_buttons_fieldset` with the `shared/radio_button_set` partial. I'm sure it depends on the scenario.